docs/reference/commandLine.md

```diff
@@ -17,54 +17,54 @@ spark-submit --master yarn --deploy-mode client --class io.smartdatalake.app.Def
   -c, --config <file1>[,<file2>...]
-    One or multiple configuration files or directories containing configuration files, separated by comma.
-    Entries must be valid Hadoop URIs or a special URI with scheme "cp" which is treated as classpath entry.
-  --partition-values <partitionColName>=<partitionValue>[,<partitionValue>,...]
-    Partition values to process for one single partition column.
-  --multi-partition-values <partitionColName1>=<partitionValue>,<partitionColName2>=<partitionValue>[;<partitionColName1>=<partitionValue>,<partitionColName2>=<partitionValue>;...]
-    Partition values to process for multiple partition columns.
-  -s, --streaming
-    Enable streaming mode for continuous processing.
-  --parallelism <int>
-    Parallelism for DAG run.
-  --state-path <path>
-    Path to save run state files. Must be set to enable recovery in case of failures.
-  --override-jars <jar1>[,<jar2>...]
-    Comma separated list of jar filenames for child-first class loader. The jars must be present in classpath.
-  --test <config|dry-run>
-    Run in test mode: config -> validate configuration, dry-run -> execute prepare- and init-phase only to check environment and spark lineage
-  --help
-    Display the help text.
-  --version
-    Display version information.
+                           One or multiple configuration files or directories containing configuration files, separated by comma. Entries must be valid Hadoop URIs or a special URI with scheme "cp" which is treated as classpath entry.
+  -o, --config-value-overwrite <nested.key>=<value>
+                           Overwrite configuration value at given nested key. Note that it is not recommended to overwrite array values. Use overwrite together with hocon substitution for this.
+  -p, --partition-values <partitionColName>=<partitionValue>[,<partitionValue>,...]
+                           Partition values to process for one single partition column.
+  -m, --multi-partition-values <partitionColName1>=<partitionValue>,<partitionColName2>=<partitionValue>[;(<partitionColName1>=<partitionValue>,<partitionColName2>=<partitionValue>;...]
+                           Partition values to process for multiple partition columns.
+  -s, --streaming          Enable streaming mode for continuous processing.
+  --parallelism <int>      Max number of parallel executed SDLB actions.
+  --state-path <path>      Hadoop path to save run state files. Must be set to enable recovery in case of failures.
+  --test <config|dry-run>  Run in test mode: config -> validate configuration, dry-run -> execute prepare- and init-phase only to check environment and spark lineage
+  --help                   Display the help text.
+  --version                Display version information.
 ```
 
+The **DefaultSmartDataLakeBuilder** class should be fine in most situations. 
+If a Spark Session needs to be created, it will use the default for Spark master, e.g. `local[*]`.
```

Comment:
```suggestion
It tries to use an existing Spark session of the environment, e.g. Databricks Cluster. It will not create a new Spark session. See SparkSmartDataLakeBuilder below to create a new Spark session.
```